package.json

```diff
@@ -7,7 +7,7 @@
     "url": "git://github.com/ulfryk/angular-typescript.git"
   },
   "scripts": {
-    "postinstall": "bower install",
+    "postinstall": "bower install && typings install",
```

Comment:
Good call on this! I didn't even know you could set up this kind of a hook for `npm` :)

Really nice that you can get all dependencies in one fell swoop with just the `npm install` command.